KmsClient.js

```diff
@@ -10,13 +10,21 @@ import {signCapabilityInvocation} from 'http-signature-zcap-invoke';
 const SECURITY_CONTEXT_V2_URL = 'https://w3id.org/security/v2';
 const DEFAULT_HEADERS = {Accept: 'application/ld+json, application/json'};
 
+/**
+ * A node https module.
+ *
+ * @typedef {object} https
+ * @see https://nodejs.org/docs/latest-v11.x/api/https.html
+ */
+
 export class KmsClient {
   /**
    * Creates a new KmsClient.
    *
-   * @param {Object} options - The options to use.
-   * @param {string} [keystore=undefined] the ID of the keystore that must be a
-   *   URL that refers to the keystore's root storage location; if not given,
+   * @param {object} options - The options to use.
+   * @param {string} [options.keystore=undefined] - The ID of the keystore
+   * that must be a URL that refers to the keystore's root storage
+   * location; if not given,
```

Comment:
Fix spacing (indent 2).

Reply:
Addressed in https://github.com/digitalbazaar/web-kms-client/pull/2/commits/09d4ec93574132f2a7a5205559256f63a81ca903